app/uk/gov/hmrc/agentservicesaccount/views/pages/manage_account.scala.html

```diff
@@ -22,7 +22,7 @@
 
 @this(mainTemplate: main,
         secondaryNav: secondary_nav,
-        h1: h1, h2: h2, h3: h3, p: p, a: a, span: span,
+        h1: h1, h2: h2, h3: h3, p: p, a: a, span: span, link_as_button: link_as_button,
```

Comment:
Does the link_as_button component have the "data-module" component?

https://jira.tools.tax.service.gov.uk/browse/APB-6451

Reply:
why of course it does!